draft-ietf-rats-architecture.md

```diff
@@ -787,16 +787,25 @@ in order to bootstrap the sequence.
 The Verifier trusts (or more specifically, the Verifier's security
 policy is written in a way that configures the Verifier to trust) a
 manufacturer, or the manufacturer's hardware, so as to be able to
-appraise the trustworthiness of that manufacturer's devices.  In solutions
-with weaker security, a Verifier might be configured to implicitly
-trust firmware or even software (e.g., a hypervisor).  That is, it
-might appraise the trustworthiness of an application component, operating
+appraise the trustworthiness of that manufacturer's devices.
+
+In solutions with weaker security, a Verifier might be configured to implicitly
+trust firmware or even software (e.g., a hypervisor).
+That is, it might appraise the trustworthiness of an application component, operating
 system component, or service under the assumption that information
-provided about it by the lower-layer hypervisor or firmware is true.
+provided about it by the lower-layer firmware or software is true.
 A stronger level of assurance of security comes when information can be vouched
 for by hardware or by ROM code, especially if such hardware is
-physically resistant to hardware tampering.  The component that is
-implicitly trusted is often referred to as a Root of Trust.
+physically resistant to hardware tampering.
+The component that is implicitly trusted is often referred to as a Root of Trust.
+
+The manufacturer of the Attester arranges for its Attesting Environment to be provisioned with key material.
+
+The signing key material is typically in the form of an asymmetric key pair (e.g., an RSA or ECDSA private key
+and a manufacturer signed IDevID certificate) secured in the Attester.
+
+The Verifier is provided with an appropriate trust anchor, or provided with a database of public keys (rather than certificates), or even carefully secured lists of symmetric keys.
+The nature of how the Verifier manages to validate the signatures produced by the Attester is critical to the secure operation an Attestation system, but is not the subject of standardization within this architecture.
```

Comment:
```suggestion
The nature of how the Verifier manages to validate the signatures produced by the Attester is critical to the secure operation an Attestation system, but is not the subject of standardization within this architecture.
In remote attestation procedures, a Verifier comes to trust a Attester with the help of Endorsers. 
The role of Endorser is often taken on by an Attester's manufacturer.
```

Reply:
Proposed line 834 is not quite correct, and the text up in 791-797 was intended to replace that.